resources/js/electron-builder.js

```diff
@@ -16,25 +16,44 @@ const isArm64 = process.argv.includes('--arm64');
 const isWindows = process.argv.includes('--win');
 const isLinux = process.argv.includes('--linux');
 const isDarwin = process.argv.includes('--mac');
-let targetOs = 'mac';
 let phpBinaryFilename = 'php';
-if (isWindows) {
+// Default to the current platform for develop mode. Build will pass in an arg that overrides this
+let targetOs = process.platform;
+
+switch (targetOs) {
+    case 'win32':
+        targetOs = 'win';
+        break;
+    case 'darwin':
+        targetOs = 'mac';
+        break;
+    case 'linux':
+        targetOs = 'linux';
+        break;
+    default:
+        throw new Error('Unsupported platform: ' + targetOs);
+        break;
+}
+
+if (isWindows || targetOs == 'win') {
     targetOs = 'win';
     phpBinaryFilename += '.exe';
-}
-if (isLinux) {
+} else if (isLinux) {
```

Comment:
These else-if blocks seem redundant now (not the first one as it's setting the binary filename)

Reply:
The switch normalizes the `targetOs` from `process.platform`, the if statements handle the override flags passed in during build stages.

Reply:
@chrisreedio Just note when it is Linux, but also Win32... at the same time. (Ubuntu in WSL in Windows)

https://github.com/NativePHP/electron/issues/50

Reply:
I don't believe you'd get that as the process.platform will always be the actively running platform. So if running it in WSL, it'd be Linux and if running in powershell (et al) it'd be win32 which gets converted to 'win' for the bin directory lookup.

Reply:
I run WSL in Windows 10 and have been working with NativePHP from Windows itself. In the long run, I'd prefer to work in WSL and build for Windows. 

Note that only Windows 11 allows graphical WSL (linux apps). If in windows 10, the native:develop command has to be run from Windows (not WSL (for now)).

Reply:
Could you rename the variables in this case?
I think isWindows, isLinux etc is confusing. It's more like "forcedOS" right? Maybe we can simplify this a bit.